### PR TITLE
Ma/guide release oct

### PR DESCRIPTION
**Pull request by Maria-Alekseeva**

### CODE REVIEW

File: source/guide/genestack-features.rst [hunk 1]
```diff
@@ -95,29 +95,26 @@ to start initialization (for more information, please, see the section **Using G
...
+options of applications being included in the pipeline (learn more about data flows from
 the corresponding sections in **Using Genestack**). Indeed, before the initialization has begun,
 you can change the parameters, replace source files, or add more files to
-the pipeline. In practice this  means that uninitialized files are cheap and quick to
-create, can be reconfigured, used as inputs to applications to create
-other files, and later computed all at once. Remember – you always need
-to initialize your files to view your results (same rule goes for data
-flows). However, once you do, any further changes become impossible.
+the pipeline. Remember – you always need to initialize your files to view your results.
+However, once you do, any further changes become impossible.
```
Comment (daniel-ohayon) [inline source/guide/genestack-features.rst]: any further changes to the files' configuration

File: source/guide/genestack-features.rst [hunk 2]
```diff
@@ -95,29 +95,26 @@ to start initialization (for more information, please, see the section **Using G
...
 
-Furthermore, in our system you can also use existing public data flows allowing to either check quality
-of data or perform some comprehensive bioinformatic analysis, namely test
-differential expression, analyse methylation pattern of DNA, identify and characterise genetic
-variations etc.
+Furthermore, in our system you can also use existing public data flows allowing to assess the data quality
+or perform some comprehensive bioinformatic analysis, such as
+gene expression profiling and analysis across groups of samples, analysis of DNA methylation
+pattern or detection and characterisation of genetic variations etc.
```
Comment (daniel-ohayon) [inline source/guide/genestack-features.rst]: pattern*s*

File: source/guide/pipelines-rawQC-preprocessing.rst [hunk 3]
```diff
@@ -45,71 +44,34 @@ preprocessing.
...
-"My Datasets" folder.
-
-.. image:: images/my_datasets_FM.png
-
 
-To explore one of these reports, click on its name and select the FastQC Report application from the
-context menu.
+The application generates a separate FastQC Report for each sample from a tested dataset;
+you can find them in the folder "Created files". You can also explore all of them simultaneously
+with Multiple QC Report app. To do so, go to "My datasets" folder, select the created
```
Comment (daniel-ohayon) [inline source/guide/pipelines-rawQC-preprocessing.rst]: with *the* Multiple QC Report

File: source/guide/pipelines-rawQC-preprocessing.rst [hunk 4]
```diff
@@ -45,71 +44,34 @@ preprocessing.
...
-
-.. image:: images/fastqc_page_source_files.png
-
-Finally, the results can be viewed in the "Reports" section. Here you will find
-various graphs that visualize the quality of your data. We will go through all
+Fast QC Report contains various graphs that visualize the quality of your data. We will go through all
 of them one by one and tell you:
 
-1. How they should  look for data of perfect quality; 
+1. How they should look for data of perfect quality;
```
Comment (daniel-ohayon) [inline source/guide/pipelines-rawQC-preprocessing.rst]: for good-quality data

File: source/guide/pipelines-rawQC-preprocessing.rst [hunk 5]
```diff
@@ -316,18 +278,18 @@ Filter duplicated reads
...
-
-.. _Tally: http://www.ebi.ac.uk/~stijn/reaper/tally.html
-
-If you suspect contamination with primers, or some  other repetitive sequence.
+If you suspect contamination with primers or some other repetitive sequence.
 This should be evident from the "Sequence duplication levels" and the "Overrepresented
 Sequences" modules of the FastQC report. Keep in mind this application should not be used with
 RNA-seq data as it will remove observed differences in expression level.
 
+This tool is based on the Tally_.
```
Comment (daniel-ohayon) [inline source/guide/pipelines-rawQC-preprocessing.rst]: on Tally (no `the`)

File: source/guide/pipelines-rawQC-preprocessing.rst [hunk 6]
```diff
@@ -337,13 +299,13 @@ quality score distribution for each read.
 
 **Command line options**:
 
-1. **Minimum quality score (Phred+33 range, 0... 41)** is quality cuttoff
+1. **Minimum quality score (Phred+33 range, 0... 41)** is quality cutoff
```
Comment (daniel-ohayon) [inline source/guide/pipelines-rawQC-preprocessing.rst]: *a* quality cutoff value

File: source/guide/using-genestack-export_include.rst [hunk 7]
```diff
@@ -3,29 +3,30 @@
...
+
 
 The application creates an Export File that contains a special link to download
 the selected assays, that is accessible until the corresponding Export File is
-deleted.
-
-.. image:: images/export3.png
-
-**Sharing the link** enables your collaborator to download data even if they
+deleted. **Sharing the link** enables your collaborator to download data even if they
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-export_include.rst]: collaborators

File: source/guide/using-genestack-import_include.rst [hunk 8]
```diff
@@ -284,26 +301,18 @@ For instance, in this case we added new column
...
 stored on Genestack, so later you can download them from the platform, in case
 they get lost on your computer.
 
-**How to upload an attachment?**
-
-Attachments should be uploaded together with the dataset. In the the Import application, choose
-the attachments from your computer along with your dataset. The platform will
-recognize the raw data, and all additional files that were unrecognised will
-be added to the dataset as attachments.
+You can import an attachment during data importing process in the the Data Import application. To do
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-import_include.rst]: the data importing process

File: source/guide/using-genestack-public-data_include.rst [hunk 9]
```diff
@@ -15,35 +15,39 @@ To prepare these Dictionaries we adopted terms from external ontologies or creat
...
 
-Our **Edit Metainfo** application enable you to prepare metadata manually. You can edit metadata on
-the last step of data importing process (see Import section for more information) or
-later from any place of the platform with context menu.
-
-Regardless the way we access Edit Metainfo application on its page we can see that
-our files are broken down into groups by file type and their metadata are shown in tables where
-rows represents metainfo fields.
+**Metainfo Editor** application enable you to explore metadata for datasets or standalone
+files. Besides, if you have enough permissions, you can edit metainfo data or import it from
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-public-data_include.rst]: metainfo data => metadata

File: source/guide/using-genestack-public-data_include.rst [hunk 10]
```diff
@@ -15,35 +15,39 @@ To prepare these Dictionaries we adopted terms from external ontologies or creat
...
+spreadsheet in .xls, .xlsx, .csv formats. You can access Metainfo Editor from any place of
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-public-data_include.rst]: you can access the metainfo editor from anywhere in the platform via the context menu

File: source/guide/using-genestack-public-data_include.rst [hunk 11]
```diff
@@ -15,35 +15,39 @@ To prepare these Dictionaries we adopted terms from external ontologies or creat
...
+the platform with context menu. Moreover, metadata editing is the last step in the data importing
+process (see Import section for more information). Metadata of the files are shown in Excel-like
+tables where rows represent metainfo fields, such as 'Organism', 'Cell line' or 'Platform'.
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-public-data_include.rst]: *columns*, not rows

File: source/guide/using-genestack-public-data_include.rst [hunk 12]
```diff
@@ -15,35 +15,39 @@ To prepare these Dictionaries we adopted terms from external ontologies or creat
...
 .. image:: images/metainfo-editor.png
 
-By default a table is based on **Default Import Template** that, however, you
+1. **Edit metadata manually**
+
+By default a metainfo data table is based on **Default Import Template** that, however, you
 can easily replace with a custom one (learn more about templates in the section
 Importing data). To do so click on the template's name, select **Change
-template**, then in the appear pop-up window pick template of interest.
+template**, and in the appear pop-up window pick template of interest.
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-public-data_include.rst]: and select the template you want in the popup window.

File: source/guide/using-genestack-public-data_include.rst [hunk 13]
```diff
@@ -52,34 +56,57 @@ example, for free-text values you should select "Text", and for numeric value
...
 them. Click **Apply naming scheme** button and select metainfo fields that you
 want to use to create names.
 
 .. image:: images/naming-scheme.png
+   :scale: 70 %
+   :align: center
+
+4. **Make a subset**
+
+In case when you want to analyse just some samples from a dataset, you can make a subset.
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-public-data_include.rst]: If you just want to analyse some samples from a dataset...

File: source/guide/using-genestack-run-DF_include.rst [hunk 14]
```diff
@@ -1,66 +1,64 @@
...
+    :align: center
 
-.. image:: images/view_as_text.png
-   :scale: 75 %
-
-Clicking the **New folder with files** button will create a folder with all the
-files used in this pipeline.
+View the text description of the pipeline including all the steps.
+Click **View as text** button to see which applications, parameters and tools were used at each
+step of analysis.
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-run-DF_include.rst]: of the analysis

File: source/guide/using-genestack-task-manager_include.rst [hunk 15]
```diff
@@ -2,34 +2,34 @@ Task manager
...
 track the progress of your computations.
 
 .. image:: images/task-manager.png
 
-All your tasks can be sorted and filtered by file name, accession, status,
-owner, last update and elapsed time. You can also view logs produced for each
-computation (error and output log). Error logs tell you why your task has
+Tasks can be sorted and filtered by application name, file name, accession, status,
+a user who started tasks, last update and elapsed time. You can also view output and error
+logs produced for each computation. Error logs tell you why a task has
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-task-manager_include.rst]: for each task

File: source/guide/using-genestack-task-manager_include.rst [hunk 16]
```diff
@@ -2,34 +2,34 @@ Task manager
...
 failed. Output logs contain information about the exact details of what
 Genestack does with your files during the computation process, what specific
-tools and parameters are used, and so on.
-
-If the computations finished successfully, error logs will be empty, but the
-logs can provide you with some basic statistics about the output data.
+tools and parameters are used, and so on. If the computations finished successfully,
+error logs will be empty, but the logs can provide you with some basic statistics
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-task-manager_include.rst]: basic information about the output data

File: source/guide/using-genestack-task-manager_include.rst [hunk 17]
```diff
@@ -2,34 +2,34 @@ Task manager
...
+about the output data.
 
 .. image:: images/task-log.png
 
 If you change your mind about a computation after it has started, remember that
 you can kill tasks whenever you want by clicking the **Cancel** button, next to
-the task status.
+the task status. To rerun an analysis click file name and **Restart initialization**.
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-task-manager_include.rst]: and select...

File: source/guide/using-genestack-task-manager_include.rst [hunk 18]
```diff
@@ -2,34 +2,34 @@ Task manager
...
 
 Statuses in Task Manager help you keep track of your tasks. Let’s look what
 each status means:
 
 -  *Starting* — the computation process has started to run;
 -  *Done* — the task has finished successfully;
 -  *Failed* — the computation has failed. To find out why, click on "View logs";
--  *Queued* — the task is waiting for dependencies to complete initialization;
+-  *Queued* — the task is pending for execution, for example when it is waiting for
```
Comment (daniel-ohayon) [inline source/guide/using-genestack-task-manager_include.rst]: to complete initialization.

Comment (Maria-Alekseeva) [reply source/guide/using-genestack-task-manager_include.rst]: fixed

**Pull request merged**